test/tests/performance/pbs_sched_perf.py

```diff
@@ -446,20 +446,15 @@ def test_multi_sched_perf(self):
         for sc in self.scheds:
             a = {'scheduling': 'False'}
             self.server.manager(MGR_CMD_SET, SCHED, a, id=sc)
-        for sc in self.scheds:
-            self.scheds[sc].log_match("Leaving Scheduling Cycle",
-                                      starttime=int(start),
-                                      max_attempts=30, interval=1)
         for sc in self.scheds:
             if sc != 'default':
-                filename = self.scheds[sc].attributes['sched_log']
-                sl = PBSSchedulerLog()
-                end = time.time()
-                sl.analyze(filename, start, end, self.server.hostname)
-                cycles = sl.cycles
-                if cycles is None or len(cycles) == 0:
-                    return []
-                dur = cycles[-1:][0].end - cycles[-1:][0].start
+                self.logger.info("searching log for scheduler " + str(sc))
+                log_msg = self.scheds[sc].log_match("Leaving Scheduling Cycle",
+                                                    starttime=int(start),
+                                                    max_attempts=30)
```

Comment:
Leave max_attempts as the default just incase the scheduling cycle takes longer that 15s